Modules/Scripted/DICOMLib/DICOMSendDialog.py

```diff
@@ -89,6 +102,7 @@ def onOk(self):
     self.sendingIsInProgress = False
 
   def onCancel(self):
+    self.protocolSelectorCombobox.currentIndexChanged.disconnect()
```

Comment:
Do not keep adding and removing this connection (it is unexpected and very error-prone). Add it once, when the widget is set up.

Reply:
done